GoNature_Client/client/gui/StaticPaneMainPageClient.java

```diff
@@ -0,0 +1,8 @@
+package gui;
+
+import javafx.scene.layout.Pane;
+
+public class StaticPaneMainPageClient {

```

Comment:
refactor this name, doesn't standup to code standards
@yansokolov96 @DanGutchin